### DISCUSSION TITLE
raise ConnectionError(e, request=request)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#36198] [New to GitHub]
raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='github.com', port=443): i am getting this error please help? I don't have access to github.com, I only have access via vpn

**Comment by ossanna16** [comment]
Hey there @anacond-dev and thank you for asking a good question! I'm glad you found our community 🙂
